test/skate_web/views/layout_view_test.exs

```diff
@@ -45,20 +45,14 @@ defmodule SkateWeb.LayoutViewTest do
   end
 
   describe "record_sentry?/0" do
-    test "returns true if the :record_sentry env is true" do
-      reassign_env(:skate, :record_sentry, true)
+    test "returns true if the :record_sentry env is not nil" do
+      reassign_env(:skate, :sentry_frontend_dsn, true)
```

Comment:
**nitpick:** also, to be slightly more realistic you should probably set it to a string value rather than `true`.